src/ExpressionEvaluator/VisualBasic/Test/ExpressionCompiler/LocalsTests.vb

```diff
@@ -169,6 +169,7 @@ End Class"
   IL_0005:  castclass  ""System.IO.IOException""
   IL_000a:  ret
 }")
+                    ' $ReturnValue is suppressed since it always matches the last $ReturnValueN
                     VerifyLocal(testData, typeName, locals(1), "<>m1", "$ReturnValue2", "Method M2 returned", expectedFlags:=DkmClrCompilationResultFlags.ReadOnlyResult, expectedILOpt:=
```

Comment:
Discussed offline...it doesn't matter, because we never show "$ReturnValue" in the locals window...it gets transformed back into "FunctionX returned" for display.